backend/internal/litellm/service_coverage_test.go

```
@@ -0,0 +1,519 @@
+func TestSetHTTPClientReplacesClient(t *testing.T) {
+	service, _, _, _ := newLiteLLMTestEnv(t)
+	next := &http.Client{Timeout: time.Second}
+	service.SetHTTPClient(next)
+	if service.http != next {
+		t.Fatal("expected http client replacement")
+	}
+}
+
+func TestClientListModelsInvalidJSON(t *testing.T) {
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
+		w.Write([]byte(`{"data":`))
```

golangci-lint (2.13.2)

[error] Error return value of `w.Write` is not checked

(errcheck)